reseq_ros2/config/reseq_mk1_can.yaml

```diff
@@ -11,3 +12,38 @@ modules:
   - address: 0x13
     hasJoint: true
     hasEndEffector: false
+dimensions:
+  a: 0.1695                  # [m] Distance between the center of every module and the previous yaw joint
+  b: 0.18395                 # [m] Distance between the center of every module and the following yaw joint
+  d: 0.223                   # [m] Distance between pair of equivalent wheels
+  r_eq: 0.0465               # [m] Radius of the equivalent wheels
+  r_linear_vel:              # [m/s] Range of the linear velocity
+    - -0.27017
+    - 0.27017
+  r_inverse_radius:          # [1/m] Range of the inverse radius of curvature
+    - -1.5385
+    - 1.5385
+  r_pitch_vel:               # [LSB/s] Range of pitch velocity
+    - -183
+    - 183
+  r_head_pitch_vel:          # [LSB/s] Range of indepepndent head pitch velocity
+    - -400
+    - 400
+  r_head_yaw_vel:            # [LSB/s] Range of head yaw velocity
+    - -455
+    - 455
```

Comment:
I would split these values into two top level categories: agevar and scaler (or scaler_consts and agevar_consts if you prefer in order to have one top level object for each node. What do you think?

Reply:
Yes, it would be better